src/templates/type/moment-select.html

```diff
@@ -7,7 +7,7 @@
                         class="select"
                         ng-model="hours"
                         ng-disabled="disable"
-                        ng-options="hour as hour for hour in hourChoices">
+                        ng-options="hour as (trailingHours + hour).slice(-(trailingHours.length)) for hour in hourChoices">
```

Comment:
This is padding 0s. It's a very clever way to do it, found it on SO. You add two strings ('00' + '10') for example so it becomes ('0010') and then slice from the end. So '0010' becomes '10' and '0008' becomes '08'